### PR TITLE
Adding Severity, Analysis and Evolution group and fields.

### PR DESCRIPTION
**Pull request by geojayuu**

### CODE REVIEW

File: maps/models.py [hunk 1]
```diff
@@ -525,6 +525,45 @@ class Map(models.Model):
         related_name="population_movements_source_for",
     )
 
+    # Severity, analyses and evolution group
+    # Severity, and composite analysis block
```
Comment (stringfellow) [inline maps/models.py:547]: is this supposed to be here twice?

Comment (geojayuu) [inline maps/models.py:547]: Yes first comment shows that it's the start of a new group, second comment shows the block of fields inside that group.

Comment (stringfellow) [inline maps/models.py:547]: :+1:

**Comment by stringfellow** [comment]
:+1:

**Pull request merged**